power_grid_model_c/power_grid_model/include/power_grid_model/calculation_parameters.hpp

```diff
@@ -117,14 +110,27 @@ template <symmetry_tag sym_type> struct PowerSensorCalcParam {
 
 template <typename T>
 concept sensor_calc_param_type =
-    std::same_as<T, VoltageSensorCalcParam<symmetric_t>> || std::same_as<T, VoltageSensorCalcParam<asymmetric_t>> ||
-    std::same_as<T, PowerSensorCalcParam<symmetric_t>> || std::same_as<T, PowerSensorCalcParam<asymmetric_t>>;
+    std::derived_from<T, VoltageSensorCalcParam<symmetric_t>> ||
+    std::derived_from<T, VoltageSensorCalcParam<asymmetric_t>> ||
+    std::derived_from<T, PowerSensorCalcParam<symmetric_t>> || std::derived_from<T, PowerSensorCalcParam<asymmetric_t>>;
 
 static_assert(sensor_calc_param_type<VoltageSensorCalcParam<symmetric_t>>);
 static_assert(sensor_calc_param_type<VoltageSensorCalcParam<asymmetric_t>>);
 static_assert(sensor_calc_param_type<PowerSensorCalcParam<symmetric_t>>);
 static_assert(sensor_calc_param_type<PowerSensorCalcParam<asymmetric_t>>);
 
+struct TransformerTapRegulatorCalcParam {
+    double u_set{};
+    double u_band{};
+    DoubleComplex z_compensation{};
+    IntS status{};
+};
+template <symmetry_tag sym_type> struct TransformerTapRegulatorMathOutput {
+    using sym = sym_type;
+
+    IntS tap_pos{na_IntS};
+};
+
 // from, to side
```

Comment:
```suggestion
// from side, to side
```

Reply:
this has been in there for 2 years. resolved now 😬